library/src/main/scala/g8.scala

```diff
@@ -432,32 +432,24 @@ object G8 {
   }
 
   def interact(params: UnresolvedProperties): ResolvedProperties = {
-    params.filter(_._1 == "description").foreach { d =>
-      @scala.annotation.tailrec
-      def liner(cursor: Int, rem: Iterable[String]): Unit = {
-        if (!rem.isEmpty) {
-          val next = cursor + 1 + rem.head.length
-          if (next > 70) {
-            println()
-            liner(0, rem)
-          } else {
-            print(rem.head + " ")
-            liner(next, rem.tail)
-          }
-        }
```

Comment:
If the existing templates have been created with automatic line break in mind, wouldn't this be a breaking change? Maybe it would be better to do manual line break only when the description contains `\n`.

Reply:
Yes, that's a breaking change for that cases - and it's something I wanted to challenge for several reasons

1. Modern consoles supports word wrapping out of the box - the code above is only valuable to have a soft wrap of the whole word
2. 70 chars? Well it's not an old times with 80 chars limitation anymore, and I doubt you'll be using g8 in mobile